integration/tests/compiler_compatibility/test_solc_versions.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import solcx
+from integration.tests.compiler_compatibility.helpers.erc_20_common_checks import Erc20CommonChecks
+from utils.helpers import generate_text
+
+INIT_NAME = "SampleToken"
+INIT_SYMBOL = "ST"
+INIT_DECIMALS = 18
+INIT_SUPPLY = 1000
+
+
+def get_three_last_solc_versions():
+    return [str(item) for item in solcx.get_installable_solc_versions()[0:3]]
+
+
+class TestSolcCompatibility(Erc20CommonChecks):
+    @pytest.fixture(scope="class", params=get_three_last_solc_versions(), ids=get_three_last_solc_versions())
```

Comment:
in this case you will run get_three_last_solc_versions() 4 times.
You should do something like https://stackoverflow.com/questions/42228895/how-to-parametrize-a-pytest-fixture 
Or just save results from get_three_last_solc_versions() in global variable in module scope and use it